app/frontend/src/search/ui/autocomplete.test.js

```diff
@@ -0,0 +1,26 @@
+import { isActive, getOptions } from './autocomplete';
+
+describe('isActive', () => {
+    test('should activate autocomplete if threshold has been met', () => {
+        expect(isActive(3, 'so')).toBe(false);
```

Comment:
Just to make it explicit, would you mind moving the above test into its own stanza:

```
  test('does not trigger autocomplete if the threshold has not been met', () => {
    expect(isActive(3, 'so')).toBe(false);
  });
```